src/node.h

```diff
@@ -254,6 +254,8 @@ NODE_EXTERN MultiIsolatePlatform* CreatePlatform(
     v8::TracingController* tracing_controller);
 NODE_EXTERN void FreePlatform(MultiIsolatePlatform* platform);
 
+NODE_EXTERN MultiIsolatePlatform* GetDefaultPlatform();
```

Comment:
Poor name since it implies there is also a non-default platform.  Just `GetPlatform()`?

Reply:
I was thinking this platform is created in `node::Start` so it seems to be the default one. 
And since we also have `CreatePlatform` api, I don't want people to think they can get newly created platform with this.

Reply:
How about `GetCurrentPlatform()`?